# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: d26634229052
# Full Hash: d26634229052dc5e3f0261c2e4bf7ffc14715d52
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-12-11 21:46:29
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Added const in IdleConnectionRunnable. r=dom-workers-and-storage-reviewers,ytausky
#   
#   Depends on D56009
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56010
# ==============================================================================

diff -r 8726e5de9204 -r d26634229052 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Wed Dec 11 14:40:38 2019 +0000
+++ b/dom/indexedDB/ActorsParent.cpp	Wed Dec 11 13:25:21 2019 +0000
@@ -5079,7 +5079,7 @@
 };
 
 class ConnectionPool::IdleConnectionRunnable final : public ConnectionRunnable {
-  bool mNeedsCheckpoint;
+  const bool mNeedsCheckpoint;
 
  public:
   IdleConnectionRunnable(DatabaseInfo* aDatabaseInfo, bool aNeedsCheckpoint)
